zone/spells.cpp

```diff
@@ -7100,3 +7100,11 @@ const CombatRecord &Mob::GetCombatRecord() const
 {
 	return m_combat_record;
 }
+
+bool Mob::CheckWaterLoS(Mob* los_attacker, Mob* los_target) // checks if both attacker and target are both in or out of the water
```

Comment:
Should just have `los_target` as the only parameter and use `los_attacker` as `this` based on this being a member method.

```cpp
bool Mob::CheckWaterLos(Mob* los_target)
```

```cpp
return zone->watermap->InLiquid(GetPosition()) == zone->watermap->InLiquid(los_target->GetPosition());
```